File: stocksell/stockbuy.java

	
	public static void main(String[] args)
    {
			Scanner sc=new Scanner(System.in);
            int n = sc.nextInt(); 
			int []arr= Arrays.stream(sc.next().split("[,\\[\\]]")).filter(w-> !w.equals("")).mapToInt(Integer::parseInt).toArray();
		    Solution g=new Solution();
            int ans=g.maximum_profit(arr,n);
            System.out.println(ans);
          
	}
}
class Solution
{
	public static int maximum_profit(int prices[],int n)
        {
        
			int maxi_value =-1;
            //Write your code here without removing the existing code
	        //the variable 'arr' contains array of integers of size n.
	        //modified the variable 'maxi_value' contain the output of the program in integer format.
            return maxi_value;
        }
}

File: stocksell/stockbuy.py
# Starter Code
class Solution:
    def maximum_profit(self,prices, n):
       maxi_value =-1
       #Write your code here without removing the existing code
	   #the variable 'arr' contains array of integers of size n.
	   #modified the variable 'maxi_value' contain the output of the program in integer format.
       return maxi_value


if __name__ == '__main__':
    n = int(input())
    str = input()
    arr = [int(int(i)) for i in str.strip('[]').split(',')]
    x = Solution()
    ans = x.maximum_profit(arr, n)
    print(ans)

